core/os/time.cpp

```diff
@@ -382,10 +382,10 @@ String Time::get_time_string_from_system(bool p_utc) const {
 
 Dictionary Time::get_time_zone_from_system() const {
 	OS::TimeZoneInfo info = OS::get_singleton()->get_time_zone_info();
-	Dictionary timezone;
-	timezone["bias"] = info.bias;
-	timezone["name"] = info.name;
-	return timezone;
+	Dictionary ret_timezone;
```

Comment:
This and `daylight` are caused by global names

Reply:
They appear to only occur on mingw in my testing so not sure if any special treatment should be given to them instead